src/styles/utils/_vars.scss

```diff
@@ -1 +1,8 @@
-$c-gray: #eee;
+$page-backgroun-color:#e4dfd3;
```

Comment:
There is a typo in the variable name. It should be $page-background-color instead of $page-backgroun-color.